stdlib/tkinter/ttk.pyi

```diff
@@ -1012,7 +1012,7 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
         anchor: tkinter._Anchor = ...,
         command: str | Callable[[], Any] = ...,
     ) -> _TreeviewHeaderDict | None: ...
-    def identify(self, component, x, y): ...
+    def identify(self, component: Literal["region", "item", "column", "row", "element"], x: int | str, y: int | str) -> str: ...
```

Comment:
This `identify` method is supposed to be an internal method, so it is best left untyped. You should use the `identify_foo()` methods below instead.